apps/api/src/app.module.ts

```diff
@@ -23,11 +23,10 @@ import { ConversationsModule } from './conversations/conversations.module';
         password: configService.get('POSTGRES_PASSWORD'),
         database: configService.get('POSTGRES_DATABASE'),
         autoLoadEntities: true,
-        // TODO check NODE_ENV
-        //
         // From NestJS docs:
```

Comment:
Feels like the comment can go away as well then

Reply:
I don't know, I feel like this is an important option and the warning should stay. But I can erase it if you don't think that too.

Reply:
Doesn't really matter. Let's merge